generator/js_generator.cc

```diff
@@ -97,7 +102,7 @@ bool IsReserved(const std::string& ident) {
 
 std::string GetSnakeFilename(const std::string& filename) {
   std::string snake_name = filename;
-  ReplaceCharacters(&snake_name, "/", '_');
+  absl::StrReplaceAll(snake_name, {{"/", "_"}});
```

Comment:
absl::StrReplaceAll takes a string_view rather than a string&, so you need to use the return value (here and below). I believe this is what's causing the latest build error.

Reply:
Good catch, this was (obviously) causing problems internally for us as well. Resolved with this commit: [`6350195` (#196)](https://github.com/protocolbuffers/protobuf-javascript/pull/196/commits/6350195f2ffd736d5627b74cabfd065572234273)